GitWorkflow/GitWorkflow/main.cpp

```diff
@@ -1,23 +1,73 @@
 #include <SFML/Graphics.hpp>
-
 #include "Player.hpp"
+#include "Utility.hpp"
+#include <vector>
+#include <chrono>
+
+#define WINDOW_WIDTH  (1080)
+#define WINDOW_HEIGHT (720)
+
+using Clock = std::chrono::steady_clock;
+using Time = std::chrono::time_point<Clock>;
+
+std::vector<Platform> platforms;
+sf::Vector2f platformSize = { 100.0f, 20.0f };
```

Comment:
Global works for now, just adding a note that we should create a resource manager later

Reply:
Will update in a later sprint.